### DISCUSSION TITLE
How to access the grid's css from the graphEditor

### DISCUSSION THREAD

**Discussion by TrickShotMLG02** [eckig/graph-editor#190] [Q&A] [answered]
It may be a bit of a stupid question, but how can I access and change the css of the grid in the graph editor from a function?

**Comment by eckig** [comment]
- What exactly do you want to change? The grid stroke color?
- When and how do you want to change it? At launch or dynamically at runtime?

**Reply by TrickShotMLG02** [reply]
I want to change the stroke color and eventually the stroke width of the grid at launch

**Reply by eckig** [reply, marked as answer]
This should work, if you include it in your JavaFX Stylesheet:
```
.graph-editor-grid {
	-grid-color: #333333;
}
```
